meson.build

```diff
@@ -184,35 +186,7 @@ message('Git commit rev: ' +git_commit_rev)
 conf.set_quoted('GIT_COMMIT_REV', git_commit_rev)
 
 foreach compileflags :['-DBUILD_DATE=' + build_date_time,
-                       '-DGIT_COMMIT_REV=' + conf.get('GIT_COMMIT_REV'),
-                       '-DECMD_STRIP_DEBUG',
-                       '-DECMD_REMOVE_SEDC_SUPPORT',
-                       '-DECMD_REMOVE_LATCH_FUNCTIONS',
-                       '-DECMD_REMOVE_ARRAY_FUNCTIONS',
-                       '-DECMD_REMOVE_SPY_FUNCTIONS',
-                       '-DECMD_REMOVE_REFCLOCK_FUNCTIONS',
-                       '-DECMD_REMOVE_PROCESSOR_FUNCTIONS',
-                       '-DECMD_REMOVE_GPIO_FUNCTIONS',
-                       '-DECMD_REMOVE_I2C_FUNCTIONS',
-                       '-DECMD_REMOVE_POWER_FUNCTIONS',
-                       '-DECMD_REMOVE_ADAL_FUNCTIONS',
-                       '-DECMD_REMOVE_JTAG_FUNCTIONS',
-                       '-DECMD_REMOVE_INIT_FUNCTIONS',
-                       '-DECMD_REMOVE_TRACEARRAY_FUNCTIONS ',
-                       '-DECMD_REMOVE_SENSOR_FUNCTIONS',
-                       '-DECMD_REMOVE_BLOCK_FUNCTIONS',
-                       '-DECMD_REMOVE_MPIPL_FUNCTIONS',
-                       '-DECMD_REMOVE_PNOR_FUNCTIONS',
-                       '-DECMD_REMOVE_SP_FUNCTIONS',
-                       '-DECMD_REMOVE_UNITID_FUNCTIONS',
-                       '-DCIP_REMOVE_INSTRUCTION_FUNCTIONS',
-                       '-DCIP_REMOVE_BREAKPOINT_FUNCTIONS',
-                       '-DCIP_REMOVE_VR_FUNCTIONS',
-                       '-DCIP_REMOVE_VSR_FUNCTIONS',
-                       '-DCIP_REMOVE_PORE_FUNCTIONS',
-                       '-DCIP_REMOVE_RW_FUNCTIONS',
-                       '-DCIP_REMOVE_MBOX_FUNCTIONS',
-                       '-DCIP_REMOVE_PMC_VOLTAGE_FUNCTIONS']
+                       '-DGIT_COMMIT_REV=' + conf.get('GIT_COMMIT_REV')]
```

Comment:
These changes are incomplete if you want to make use of #12

You have to place the `src/common` include path first in your include defines so it picks up the right version of ecmdDefines.H.

You also need to turn on EDBG_BUILD_EBMC so that the right compile defines are selected for you.

Reply:
Yes , I will fix it